02-spatial-data.Rmd

```diff
@@ -652,10 +660,11 @@ extent(new_raster)
 crs(new_raster)
 ```
 
-Unlike the **sf** package, **raster** only use the `proj4string` representation of the coordinate reference system.
+<!--CRSargs(CRS("+init=epsg:4326"))-->
+Note that in contrast to the **sf** package, **raster** only accepts the `proj4string` representation of the coordinate reference system.
 
 Values of the cells are the only non-spatial information in the `Raster*` object.
```

Comment:
I think @Nowosad may be saying that you must spell-out the entire proj4 string, rather than just executing `st_crs(4326)`. I suggest this sentence is rephrased in any case.